lab_1_classify_by_unigrams/main.py

```diff
@@ -34,6 +42,18 @@ def calculate_frequencies(tokens: list[str] | None) -> dict[str, float] | None:
 
     In case of corrupt input arguments, None is returned
     """
+    if not isinstance(tokens, list):
+        return None
+    freq = {}
+    for i in tokens:
+        if not isinstance(i, str):
+            return None
+        if len(i) != 1:
+            return None
```

Comment:
why do you need this?

Reply:
because of mypy

Reply:
the same goes for the following comment

Reply:
and I don't understand what "no" means in the latter